data/rhea_db/meta.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: rhea_db
+description: |-
+    Rhea is a freely available, manually annotated database of chemical reactions,
+    created by the Swiss Institute of Bioinformatics (SIB) and used as source
+    of chemical reactions for the UniProt Knowledgebase (UniProtKB) annotation.
+    This dataset contains the reactions from Rhea database and their components
+    in smiles format and ChEBI IDs.
+    Tasks can be formulated as translation between smiles and english and vice
+    versa, or as a completion of stechiometric coefficients or masked reactants.
+targets:
+    - id:
+      description:
+      units:
+      type:
+identifiers:
+    - id: rhea_id
+      type: int
+      description: Rhea reaction ID
+    - id: equation
+      type: string
+      description: reaction equation
```

Comment:
Thanks for contributing this, I didn't know this dataset. Can you post an example of a reaction equation in this dataset?

Reply:
Thanks, that helps a lot! So it is always just the name of the compounds, and they have no SMILES or some other representation?

Reply:
they do have accompanying smiles for every chemical compound name, see the
json row above ^^^.